src/ext/syntacticsearch/funcFunction.ml

```diff
@@ -255,7 +259,7 @@ let find_usesvar_in_fun funname funid funstrucname varname file =
   | Some fundec ->
       let result = ref [] in
       let dedup =
-        delete_duplicates (Hashtbl.find_all varnameMapping varname) []
+        delete_duplicates (find_all_orignames varname) []
```

Comment:
Does this work the right way around? Previously `varnameMapping` had original names as keys and new names as values:
```ocaml
H.add varnameMapping origname newname
```
So it was finding all new names for an original name.

But now the new function is called `find_all_orignames` as if it's finding all original names for a new name. Or maybe it's just the function that's confusingly named?